sotodlib/tod_ops/jumps.py

```diff
@@ -223,13 +212,11 @@ def _fix(i, jump_ranges, heights, x_fixed):
     heights = cast(NDArray[np.floating], heights)
 
     nfuture = min(len(x_fixed), NFUTURE)
-    slices = [slice(i * nfuture, (i + 1) * nfuture) for i in range(nfuture)]
+    slice_size = len(x_fixed) // nfuture
+    slices = [slice(i * slice_size, (i + 1) * slice_size) for i in range(nfuture)]
     slices[-1] = slice(slices[-1].start, len(x_fixed))
```

Comment:
No strong reason for slices over lists here, think that was just because I copied some old code here.
Agreed that sharing the overhead would be better but I will be getting rid of these futures all together soon in favor of OMP in a C++ based fixer anyways.

The lore here is that when I first added futures I did a proc pool and before I PRd them I noticed that preproc was doing proc based parallelism so I switched to a thread pool, because of GIL this really only helps magic away a small amount of overhead (waiting for mem alloc basically) so its not that useful. So I will be getting rid of it soon anyways.

Reply:
Sounds good!